pkgs/ffigen/lib/src/config_provider/config_types.dart

```diff
@@ -432,3 +433,16 @@ class Declaration {
     required this.originalName,
   });
 }
+
+class ExternalVersions {
+  final Versions? ios;
+  final Versions? macos;
+  const ExternalVersions({this.ios, this.macos});
+}
+
+class Versions {
+  final Version? min;
+  final Version? max;
+  final Version? target;
```

Comment:
As in other comments, maybe `target` only makes sense for apps and not for bindings? So maybe remove it. (I'm also learning as we go here. 🤓)

Reply:
Done